precompiles/src/solidity/codec/xcm.rs

```diff
@@ -76,6 +76,7 @@ pub(crate) fn network_id_to_bytes(network_id: Option<NetworkId>) -> Vec<u8> {
 			encoded.append(&mut block_hash.into());
 			encoded
 		}
+		/*
```

Comment:
This code commented can be removed entirely.

Reply:
Addressed in https://github.com/polkadot-evm/frontier/pull/1614/commits/1217ef1da183345d93ec57e2638ea668cc968cf6